tests/utils/handler/cluster_handler.go

```diff
@@ -577,6 +577,13 @@ func (ch *clusterHandler) GenerateClusterCreateFlags() ([]string, error) {
 			ch.clusterConfig.Subnets = &ClusterConfigure.Subnets{
 				PrivateSubnetIds: strings.Join(subnets["private"], ","),
 			}
+			if !ch.profile.ClusterConfig.HCP {
```

Comment:
Please make sure it is 4.19 version. And better to have an environment exposed so it is easy when you trigger other versions test.